ci/test_cpp_common.sh

```
@@ -9,6 +9,9 @@ set -euo pipefail
 rapids-logger "Configuring conda strict channel priority"
 conda config --set channel_priority strict
 
+# TODO: Remove before merging. Use rapidsmpf conda packages from rapidsai/rapidsmpf#1081.
+source ./ci/use_conda_packages_from_prs.sh
```

Shellcheck (0.11.0)

[info] Not following: ./ci/use_conda_packages_from_prs.sh was not specified as input (see shellcheck -x).

(SC1091)